Documentation/bootstrapping-guidelines.md

```diff
@@ -0,0 +1,96 @@
+
+To build .NET SDK from source requires previously source-built .NET SDK and toolset which is also referred as bootstrapping .NET SDK and toolset. Check quick [instructions](https://github.com/dotnet/installer/blob/main/README.md#build-net-from-source-source-build) to build SDK using source-build. This document provides guidance for getting or building  bootstrapping .NET SDK and toolset in different scenarios (covering step 2 and step 3 of source-build instructions).
+
+## Building New .NET Version
+Building New .NET Version is two stage process:  
+
+**Stage 1:**  
+Build bootstrapping .NET SDK and toolset for the targeted platform.
+- prep.sh (args???)  
+ This downloads platform-native version of .NET SDK and toolset. So if you are building on x64 machine, you get x64.
+- build.sh (args???)  
+Create source built .NET SDK
+
+**Stage 2:**  
+Use source built .NET SDK created in stage 1 to build .NET SDK from source. No need to run prep.sh in this stage.
```

Comment:
You should include details on where the SDK is located? Should I copy it to a new directory? Do I need to clean the repo? Do I need to `git checkout` changed files?

Reply:
@crummel Are you able to help adding this information?

Reply:
I added some detail around this.